source/creating_content/01_creating_content_page/writing_page.rst

```diff
@@ -0,0 +1,35 @@
+Writing a content page
+======================
+
+.. admonition:: Important
+   :class: important
+
+   In the page header, next to the title of the page, are displayed language buttons -one for each language available in the domain. They allow multilingual content management. In case a content page has not been translated for a chosen language, when navigating from the front office, the page will remain blank for that chosen language.
```

Comment:
> In case a content page has not been translated for a chosen language [...] the page will remain blank for that chosen language.

Hmmm not really: When we load a template, it loads it for all languages. And by default when we start editing a new page, we load the `Custom` template. So content for other languages isn't empty, it contains the base template.

Reply:
Yes, this is indeed technically true. However, I think that it is preferable for our users to have the easiest explanation. If they land on a page which has a content in FR but not in EN, they will consider that the EN version is blank. I don't think that it will make much difference for them if we explain that technically the page is not empty, because it contains a box, but that the box it indeed empty 🤓